README.md

```diff
@@ -2,6 +2,14 @@
 
 OAI-PMH harvester for the Pacific Rim Library.
 
+## Dependencies
+
+There are a couple of dependencies a developer must have installed in order to build this project.
+
+* A [JDK](https://adoptium.net/marketplace/) (&gt;=17)
+* [Maven](https://maven.apache.org/download.cgi) (&gt;=3.8)
+* [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) (&gt;=16)
```

Comment:
[The VueJS docs specify the lowest acceptable Node version](https://vuejs.org/guide/quick-start.html#creating-a-vue-application), not NPM.

Reply:
Kept npm mention, too, since it's possible to use Node.js without npm and npm is what we call from the POM.